src/Cake.Common/Net/HttpAliases.cs

```diff
@@ -197,7 +197,7 @@ public static void DownloadFile(this ICakeContext context, Uri address, FilePath
             // We track the last posted value since the event seems to fire many times for the same value.
             var percentComplete = 0;
 
-            using (var http = new HttpClient())
+            using (var http = new HttpClient(new HttpClientHandler { UseDefaultCredentials = settings.UseDefaultCredentials }))
```

Comment:
@Rookian yes, I understand that.  However, the DownloadFile alias currently has the ability to specify a username and password for the download to happen.  The change that you are suggesting initializes the HttpClient with UseDefaultCredentials.  My question is, will the specified username and password still work, as a result of the initialization code that you have put in place?